<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCocktailWindow</class>
 <widget class="QWidget" name="AddCocktailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cocktail hinzuf√ºgen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cocktail Name:</string>
       </property>
       <property name="buddy">
        <cstring>cocktailnameedit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="cocktailnameedit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Zutat 1:</string>
       </property>
       <property name="buddy">
        <cstring>zutatenbox1</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Zutat 2:</string>
       </property>
       <property name="buddy">
        <cstring>zutatenbox2</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Zutat 3:</string>
       </property>
       <property name="buddy">
        <cstring>zutatenbox3</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Zutat 4:</string>
       </property>
       <property name="buddy">
        <cstring>zutatenbox4</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Zutat 5:</string>
       </property>
       <property name="buddy">
        <cstring>zutatenbox5</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Zutat 6:</string>
       </property>
       <property name="buddy">
        <cstring>zutatenbox6</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="zutatenbox1"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="zutatenbox2"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="zutatenbox3"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="zutatenbox4"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="zutatenbox5"/>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="zutatenbox6"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="AbortButton">
       <property name="text">
        <string>Abbrechen</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="SaveButton">
       <property name="text">
        <string>Speichern</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
